courses/free-courses-hi.md

```diff
@@ -569,6 +569,7 @@
 ### Operating Systems
 
 * [3.2 Operating System (Complete Playlist)](https://www.youtube.com/playlist?list=PLmXKhU9FNesSFvj6gASuWmQd23Ul5omtD) - Knowledge Gate
+* [Operating System](https://www.youtube.com/playlist?list=PLBlnK6fEyqRiVhbXDGLXDk_OQAeuVcp2O) - Neso Academy
```

Comment:
Consider moving to courses/free-courses-en.md

Reply:
Once you have made the requested changes, please update the PR title to include “(changes made)” and a reviewer will take another look as soon as they are available. Thanks!